webui/react/src/components/kit/Pagination.tsx

```diff
@@ -8,7 +8,7 @@ interface PaginationProps {
     type: 'page' | 'prev' | 'next' | 'jump-prev' | 'jump-next',
     originalElement: ReactNode,
   ) => ReactNode;
-  onChange?: () => void;
+  onChange?: (page: number) => void;
```

Comment:
should this be `((page: number, pageSize: number) => void)` if we wanna respect the antd type?

<img width="1000" alt="image" src="https://user-images.githubusercontent.com/109113616/211632462-2b94ead0-a7e9-4c3d-9628-67d8aa08e772.png">